analysis/src/Scope.ml

```diff
@@ -1,13 +1,9 @@
-type item =
-  | Constructor of string * Location.t
-  | Field of string * Location.t
-  | Module of string * Location.t
-  | Open of string list
-  | Type of string * Location.t
-  | Value of string * Location.t * SharedTypes.Completable.contextPath option
+type item = SharedTypes.ScopeTypes.item
 
 type t = item list
 
+open SharedTypes.ScopeTypes
```

Comment:
where is this used?

Reply:
Ah in the pattern matching below.

Reply:
Yeah exactly, I opened it just to have it function as it did before.